src/System.ServiceModel.Syndication/ref/System.ServiceModel.Syndication.csproj

```diff
@@ -6,6 +6,15 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.ServiceModel.Syndication.cs" />
+    <Compile Include="..\src\System\ServiceModel\Syndication\Atom10Constants.cs">
```

Comment:
We do not include sources into the reference assembly. Instead, regenerate the reference assembly using `/t:GenerateReferenceSource`

Reply:
Yeah, here is a place where we actually want duplication.  We don't want the source to change and automatically update the ref.  We need people to stop and think about it when their change impacts the reference assembly source.

Reply:
Will change.